Readme.md

```diff
@@ -3,11 +3,11 @@
 <h5 align="center">Fast, flexible & lean implementation of core jQuery designed specifically for the server.</h5>
 
 <div align="center">
-  <a href="http://travis-ci.org/cheeriojs/cheerio">
-    <img src="https://secure.travis-ci.org/cheeriojs/cheerio.svg?branch=master" alt="Travis CI" />
+  <a href="https://travis-ci.org/cheeriojs/cheerio">
+    <img src="https://travis-ci.org/cheeriojs/cheerio.svg?branch=main" alt="Travis CI" />
   </a>
   <a href="https://coveralls.io/r/cheeriojs/cheerio">
-    <img src="http://img.shields.io/coveralls/cheeriojs/cheerio.svg?branch=master&style=flat" alt="Coverage" />
+    <img src="https://img.shields.io/coveralls/cheeriojs/cheerio.svg?branch=main&style=flat" alt="Coverage" />
```

Comment:
coveralls doesn't seem to be integrated anymore. Should I drop the badge? Alternatively, we could bring coveralls back in #1600 later.

Reply:
I had to reset coveralls recently. I hope this will populate again soon.